Dockerfile

```
@@ -151,19 +151,19 @@ VOLUME /tmp
 VOLUME ${ZB_HOME}/data
 VOLUME ${ZB_HOME}/logs
 
-RUN groupadd -g 1000 zeebe && \
-    adduser -u 1000 zeebe --system --ingroup zeebe && \
+RUN groupadd -g 1001 zeebe && \
+    adduser -u 1001 --system --gid 1001 -h ${ZB_HOME} zeebe && \
     chmod g=u /etc/passwd && \
     # These directories are to be mounted by users, eagerly creating them and setting ownership
     # helps to avoid potential permission issues due to default volume ownership.
     mkdir ${ZB_HOME}/data && \
     mkdir ${ZB_HOME}/logs && \
-    chown -R 1000:0 ${ZB_HOME} && \
+    chown -R 1001:0 ${ZB_HOME} && \
     chmod -R 0775 ${ZB_HOME}
 
-COPY --link --chown=1000:0 docker/utils/startup.sh /usr/local/bin/startup.sh
-COPY --from=dist --chown=1000:0 /zeebe/camunda-zeebe ${ZB_HOME}
+COPY --link --chown=1001:0 docker/utils/startup.sh /usr/local/bin/startup.sh
+COPY --from=dist --chown=1001:0 /zeebe/camunda-zeebe ${ZB_HOME}
 
-USER zeebe:zeebe
+USER 1001:1001
```

If I get you correctly, all that matters would be to use the numeric ids for the default user, wouldn't this be enough then, making this non-breaking?
```suggestion
USER 1000:1000
```

---

We also need to unify the ID across the apps because it requires a lot of changes in Helm and also confusing for end users (we have like 10 apps everyone has its own id and it's not clear which is which).

---

thanks for clarifying!